src/makeInfinityAdder.js

```diff
@@ -3,8 +3,24 @@
 /***
  * @return {function}
  */
-function makeInfinityAdder() {
-  // write code here
+function makeInfinityAdder(a) {
```

Comment:
The parameter 'a' is defined but never used. Since the function `makeInfinityAdder` does not need any initial parameters according to the task description, you should remove it to avoid confusion.